www/CQMContext.js

```diff
@@ -311,4 +323,94 @@ CQMContext.prototype._getEntitlementInfo = function(entityNames, successCallback
     exec(successCallback, fail, "CQMContext", "getEntitlementInfo", [entityNames]);
 };
 
+/**
+ * Get status for an Entity such as isSavable, progressPct, progressState. (Current only supports Collection)
+ *
+ * @param {Entity} or {EntityList} Entity we want to get status for or EntityList we want to get status for.
```

Comment:
You're missing the actual parameter name on some of the docs in this file (should it be "entityOrEntityList" here?)  "entity" and "isSilent" below, etc.